mise.toml

```
@@ -561,6 +561,27 @@ docker tag cipherstash/proxy:latest cipherstash/proxy:latest
 docker push cipherstash/proxy:latest
 """
 
+[tasks."release:aws"]
```

This is good namespacing, and we can make it more descriptive:

```suggestion
[tasks."release:aws-marketplace"]
```